aten/src/ATen/test/half_test.cpp

```diff
@@ -119,6 +119,7 @@ ASSERT_SAME_TYPE(tinyness_before);
 
 TEST(TestHalf, CommonMath) {
   float threshold = 0.00001;
+  (void)threshold;
```

Comment:
What's going on here? Is this dead code in opt mode because asserts are compiled out? Should probably not use `assert` in this test in that case